service/dap/daptest/client.go

```diff
@@ -48,6 +48,10 @@ func (c *Client) send(request dap.Message) {
 	dap.WriteProtocolMessage(c.conn, request)
 }
 
+func (c *Client) ReadMessage() (dap.Message, error) {
```

Comment:
Just a note that this will go in https://github.com/go-delve/delve/pull/1889, so you may want to add a note to the PR description mentioning that.

Reply:
I waited for your version to go in, so there are no conflicts.